harness/tests/experiment/fixtures/pytorch_onevar_model.py

```diff
@@ -407,6 +449,200 @@ def __init__(self, context: pytorch.PyTorchTrialContext) -> None:
         )
 
 
+class OneVarTrialCustomEval(BaseOneVarTrial):
+    _searcher_metric = "loss"
```

Comment:
If I am not mistaken, the name of the searcher metric refers to the metric returned by the evaluation and not by training. So this should probably be `"val_loss"` (the same name as on line 464).